packages/vite/src/types/lightningcss.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { CSSModulesConfig, Drafts, Targets } from 'lightningcss'
+
+/*
+ Using a namespace create issue once bundled because definition are nor inlined,
+ and it creates types like `export type Drafts = Drafts`
+*/
+
+export type LightningCSSModulesConfig = CSSModulesConfig
+export type LightningCSSDrafts = Drafts
+export type LightningCSSTargets = Targets
```

Comment:
Would it work if we export the types under `LightningCSS` as we do for rollup? https://github.com/vitejs/vite/blob/9a42e31295d0eb5368e68e5288fbf2f27148137c/packages/vite/src/node/index.ts#L3

Reply:
Using `*` would bundle too much types IMO.

But yeah using: 
```ts
 export type LightningCSS = {
   CSSModulesConfig: CSSModulesConfig;
   Drafts: Drafts;
   Targets: Targets; 
} 
```

You then need to do `LightningCSS["Drafts"]` which is a bit less nice but ok.

(This doesn't solve the api extractor warnings though)